Birthday-2024-Project/Scripts/Game/States/GameWinState.gd

```diff
@@ -1,20 +1,10 @@
 class_name GameWinState
 extends GameState
 
-
-func next_puzzle():
-	manager.go_to_next_level()
-
-
-func reset_puzzle():
+func _reset_puzzle():
```

Comment:
do we need to have the option to reset puzzle when we are in win state?

Reply:
The level navigations are still accessible on the win state. If the player wants to restart the level, doesn't hurt to have the option. It's inherited anyway